cmd/apiserver-boot/boot/create/resource.go

```diff
@@ -207,28 +207,33 @@ func createResource(boilerplate string) {
 	}
 
 	if !skipGenerateController {
+		config := cfgv3.New()
+		if err := config.SetMultiGroup(); err != nil {
+			klog.Fatal(err)
+		}
+		if err := config.SetRepository(util.Repo); err != nil {
+			klog.Fatal(err)
+		}
+		if err := config.SetDomain(util.Domain); err != nil {
+			klog.Fatal(err)
+		}
+
 		// write controller-runtime scaffolding templates
-		r := &resource.Resource{
-			Namespaced:  !nonNamespacedKind,
-			Group:       groupName,
-			Version:     versionName,
-			Kind:        kindName,
-			Plural:      resourceName,
-			Package:     filepath.Join(util.Repo, "pkg", "apis", groupName, versionName),
-			ImportAlias: resourceName,
```

Comment:
Curious what happened to `ImportAlias`.

Reply:
Moved [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/resource.go#L89)